nuxt.config.ts

```diff
@@ -4,27 +4,34 @@ import domains from './configs/domains'     ;
 
 const css   =   [ '@/assets/custom.scss', 'vue-final-modal/style.css' ]
 
+
 export default defineNuxtConfig({
 
   devtools: { enabled: false},
   debug: false,
   sourcemap: { server: true, client: true },
   css,
+    app:{
+    pageTransition: { name: 'page', mode: 'out-in'  }
+  },
   runtimeConfig:{
     apiUser     : process.env.API_USER,
     apiUserPass : process.env.API_USER_PASS,
     apiKey      : process.env.API_KEY,
     panoramaKey : process.env.PANORAMA_KEY,
     public: {
-      isLocalHost:!!process.env.NUXT_IS_LOCAL_HOST,
-      showBl1Link:!!process.env.NUXT_SHOW_BL1_LINK,
+      isLocalHost:process.env.NUXT_PUBLIC_IS_LOCAL_HOST==='true'?true:false,
+      showBl1Link:process.env.NUXT_PUBLIC_SHOW_BL1_LINK==='true'?true:false,
```

Comment:
for runtimeconfig you don't have to di any process.env...
nuxt auto load NUXT_  into config. not need to bind it.
you only have to set the default value

https://nuxt.com/docs/guide/going-further/runtime-config#example